### ISSUE TITLE
Unable to get branch name while trying to add changelog

### ISSUE THREAD

**Issue by ifly-leonard** [markwalet/laravel-git-state#24]
I am getting an error while trying to add changelogs. I used this package & project on a linux environment, worked well, but I had to recently change to Windows, and that's where I'm getting this issue. 

```
$ php artisan changelog:add
The system cannot find the path specified.

   MarkWalet\GitState\Exceptions\RuntimeException 

  Error while fetching the branch name

  at D:\laragon\www\PROJECT\vendor\markwalet\laravel-git-state\src\Drivers\ExecGitDriver.php:44
     40▕
     41▕         exec($command, $result, $code);
     42▕
     43▕         if ($code !== 0 || count($result) !== 1) {
  ➜  44▕             throw new RuntimeException('Error while fetching the branch name');
     45▕         }
     46▕
     47▕         return $result[0];
     48▕     }

  1   D:\laragon\www\PROJECT\vendor\markwalet\laravel-changelog\src\Commands\ChangelogAddCommand.php:90
      MarkWalet\GitState\Drivers\ExecGitDriver::currentBranch()

  2   D:\laragon\www\PROJECT\vendor\markwalet\laravel-changelog\src\Commands\ChangelogAddCommand.php:35
      MarkWalet\Changelog\Commands\ChangelogAddCommand::path(Object(MarkWalet\GitState\Drivers\ExecGitDriver))
```
Environment: 

OS: Windows 10 
PHP: 8.0.17 
Laravel: 8.83.0

Is this because of PHP 8? or OS level error?

@markwalet Any help would be appreciated :D Thank you so much.

**Comment by markwalet** [comment]
Thanks for the extra info. I transferred the issue to the correct repository :)

Not sure what the problem is yet, but I'll see if I can simulate your environment as close as possible to see if I can reproduce the problem.

If you have time, can you try to use the package on a new fresh Laravel project? With just a few commits on a single branch for example. I want to confirm that it is an environment-specific bug. And not a repository-specific bug.

**Comment by markwalet** [comment]
Fixed in v1.8.0.
